documentation/deploy-page.md

```diff
@@ -0,0 +1,10 @@
+# Deploy page
+
+Right now the offline page is a sign that is enabled via toggle, and with mvc offline.
+To see how to enable and disable the deploy page check out this [example](https://www.screencast.com/t/FizkXB8kt0iJ)
```

Comment:
If these are two different paragraphs, you should add an interline between them.

```suggestion
Right now the offline page is a sign that is enabled via toggle, and with mvc offline.

To see how to enable and disable the deploy page check out this [example](https://www.screencast.com/t/FizkXB8kt0iJ)
```